cmd/tcplb/flags.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"net"
+	"strings"
+	"tcplb/lib/core"
+)
+
+const (
+	commandName     = "tcplb"
+	upstreamListSep = ","
+)
+
+var InvalidUpstreamListFlag = errors.New("invalid upstream list flag value")
+
+// UpstreamListValue is a flag.Value for lists of Upstream addresses.
+type UpstreamListValue struct {
+	Upstreams []core.Upstream
+}
+
+func (v *UpstreamListValue) String() string {
+	n := len(v.Upstreams)
+	tokens := make([]string, n, n)
```

Comment:
Suggestion: simplify? It's equivalent.

```suggestion
	tokens := make([]string, n)
```